plugin_test.go

```diff
@@ -1,7 +1,10 @@
 package main
 
-import "testing"
-import "gotest.tools/assert"
+import (
+	"testing"
+
+	"gotest.tools/assert"
```

Comment:
also please remove this dependency.  We use the Go standard library only for testing.

Reply:
oh nevermind, it looks like someone else added this library to the repository.  This should not have slipped through code review, but we will remove this in a separate PR.

Reply:
I think the `go fmt` brings that to the code review